src/cluegetter/http.go

```diff
@@ -442,12 +466,12 @@ func httpAbusersHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	rows, err := Rdbms.Query(`
-		SELECT sender_domain, count(*) amount
```

Comment:
Even when selector == sasl_username, wouldn't you want to know the sender_domain as well? I'd suggest selecting them both.